cephprimarystorage/cephprimarystorage/imagestore.py

```
@@ -63,6 +71,31 @@ def _get_image_reference(self, path):
         return "{0}{1}/{2}".format(self.ZSTORE_PROTOSTR, name, imgid)
 
+    def storage_backup(self, cmd):
+        self._check_zstore_cli()
+        sshData = cmd.userName + '@' + cmd.hostname + ':' + cmd.uploadDir
+        cmdstr = '%s -rootca %s storagebackup -ssh %s -p %s -parent %s -volume %s -backupUuid %s -mode %s' % (
+            self.ZSTORE_CLI_PATH, self.ZSTORE_CLI_PATH, sshData, cmd.password, cmd.lastBackupUuid, cmd.volumePath,
+            cmd.backupUuid, cmd.mode)
+        output = shell.call(cmdstr.encode(encoding="utf-8"))
+        result = jsonobject.loads(output.splitlines()[-1])
+
+        _, image = self._get_id_name_from_install(cmd.volumePath)
```

Ruff (0.14.4)

Unpacked variable `image` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)